src/me/StevenLawson/TotalFreedomMod/Commands/Command_opme.java

```diff
@@ -2,10 +2,11 @@
 
 import me.StevenLawson.TotalFreedomMod.TFM_Util;
 import org.bukkit.command.Command;
+import me.StevenLawson.TotalFreedomMod.*;
```

Comment:
Why why why?????? This should NOT be needed, and the star imports from what I understand should only be used as an absolute last resort. This should not be needed either way.